src/test/java/org/folio/dew/BulkEditTest.java

```diff
@@ -727,6 +729,11 @@ private JobParameters prepareJobParameters(ExportType exportType, EntityType ent
       parametersBuilder.addLong(TOTAL_CSV_LINES, countLines(localFilesStorage, file, false), false);
     }
 
+    var tempDir = System.getProperty(TMP_DIR_PROPERTY) + PATH_SEPARATOR + springApplicationName + PATH_SEPARATOR + jobId;
```

Comment:
Plz, use Path options to create path for directory, path separator in that case will be operational system independent.